### PR TITLE
Final PR - Back End Doctors Project

### PR DESCRIPTION
**Pull request by benja27**
On this PR we:

- Understand pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end. 🔄🔗
- Understand principles of Ruby on Rails and React frameworks. 📚🛠️
- Apply Ruby best practices and language style guides in code. 💎✨
- Apply RoR best practices and language style guides in code (e.g., thin controllers). 🚀📘
- Apply JavaScript best practices and language style guides in code. 🌐📝
- Learn about and practice giving constructive feedback to teammates. 🗣️👂
- Perform a code review for a team member. 🕵️‍♂️👩‍💻
- Use the "Review change" feature from GitHub. 🌐👀
- Write clear, professional, and respectful review comments for other team members. 📝💬
- Explain "why" a change is requested when giving a code review. 🤔🔍
- Plan a 2+ week project with no interim Microverse-set milestones and submit it on time. 🗓️⏰
- Demonstrate the ability to apply best practices of communication for resolving teamwork challenges. 🤝🗣️
- Understand that respect is the foundation of strong relationship-building with teammates. 🌱❤️
- Recognize the importance of investing in good working relationships with teammates. 🤗👥
- Recognize the value of making equal contributions to group projects to produce the best outcome. 🤝💪
- Use an empty Kanban board to manage tasks with the team and own time on the project. 📊✅

### CODE REVIEW

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @benja27,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Nice code organization ✔️ 
-  API is working well✔️ 
-  Good readme ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md [hunk 1]
```diff
@@ -1,24 +1,158 @@
...
+- Navigate to config/database.yml
+- Update the user and password to match your postgresql user and password.
+
+### Create database and migrate
+Run the following code-
+
+```sh
+  rails db:create
+  rails db:migrate
+```
```
[the remark this answers is not in the document]
Comment (ClintonYounge) [reply README.md:96]: Thanks a lot for this. Forgot this step 👍

**Comment by ClintonYounge** [comment]
@Meltrust Thanks for this necessary feedback 💯 very much appreciated

File: app/controllers/doctors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    else
+      render json: @doctor.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /doctors/1
+  def destroy
+    @doctor.destroy
+    head :no_content
+  end
```
[the remark this answers is not in the document]
Comment (emarkees) [reply app/controllers/doctors_controller.rb:40]: FIXED

**Review by Meltrust — APPROVED** [review]
#### Hi @benja27,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause





Cheers and Happy coding!👏👏👏


------

**Pull request merged**